sys/net/gnrc/network_layer/ipv6/nc/gnrc_ipv6_nc.c

```diff
@@ -31,9 +31,9 @@
 #if ENABLE_DEBUG
 /* For PRIu8 etc. */
 #include <inttypes.h>
+#endif
```

Comment:
What about including `<inttypes.h>` in `debug.h`?

Reply:
nah. let's keep header dependencies as small as possible.